codebase/superdarn/src.bin/tk/tool/fit_speck_removal.1.0/fit_speck_removal.c

```diff
@@ -134,10 +134,7 @@ int main (int argc,char *argv[]) {
   }
   
   FILE *fp=NULL;
-  if (arg==argc) 
-      fp=stdin;
-  else 
-      fp=fopen(argv[arg],"r");
+  fp=fopen(argv[arg],"r");
   if (fp==NULL) {
     fprintf(stderr,"File not found.\n");
```

Comment:
@mts299 Is this better than "file not found"?
```suggestion
    fprintf(stderr,"Error opening input file.\n");
```

Reply:
How about that? ^ You can commit if you like.

Reply:
Can you seperate the two? file not found makes sense for the later